backend/database/connection.py

```
@@ -40,35 +45,26 @@ class Config:
             "connect_timeout": 30,  # Connection timeout
             "application_name": "AOM_2025_Backend"
         }
     )
-else:
-    # Use simpler engine for SQLite (no pooling or connect_args)
+elif settings.database_url.startswith("sqlite"):
+    # Use simpler engine for SQLite (development only)
+    print("⚠️  WARNING: Using SQLite for development. PostgreSQL recommended for production.")
     engine = create_engine(settings.database_url)
+else:
+    raise ValueError(f"Unsupported database URL format: {settings.database_url}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)